src/GeneralStateTestsFiller/stCreateTest/CreateTransactionCallDataFiller.yml

```diff
@@ -0,0 +1,68 @@
+CreateTransactionCallData:
+  _info:
+    comment: |
+      Tests if CALLDATALOAD, CALLDATACOPY, CODECOPY and CODESIZE work correctly in initcode context:
+      call data is always empty in initcode context and "code" is initcode.
```

Comment:
aha, you found a bug. yml text " symbol goes to json as is, breaking the format of generated test json.

Reply:
Anything I should change here?

Reply:
I pushed a fix, you can refill the tests

Reply:
I'm not able to do this because `:yul` does not work any more as it was. I'm getting a lot of weird errors.

```
Error:
Error when parsing file (/home/chfast/Projects/ethereum/tests/GeneralStateTests/stCreateTest/CreateTransactionCallData.json) Error parsing json: Dataobject array/object expected ',' when listing elements, but got `c`around:
"------
DATACOPY, CODECOPY and CODESIZE work correctly in initcode context:\ncall data is always empty in initcode context and "
"------ (GeneralStateTests/stCreateTest/CreateTransactionCallData, fork: Berlin, TrInfo: d: 2, g: 0, v: 0, TrData: `:label codecopy :yul berlin {
  // codecopy() ..`)

--------
```

And e.g. for `:yul berlin object {`:

```
yul.sh "Yul compilation error: "
Error: Call or assignment expected.
 --> /tmp/8c05-9142-2b44-ee5d:2:8:
  |
2 |   code {
  |        ^
WARNING: Reading empty result for /home/chfast/.retesteth/evmone/yul.sh /tmp/8c05-9142-2b44-ee5d berlin object "C"
Finishing retesteth run
*** Total Tests Run: 0


--------
*** TOTAL WARNINGS DETECTED: 1 warnings during all test execution!
--------
info: Reading empty result for /home/chfast/.retesteth/evmone/yul.sh /tmp/8c05-9142-2b44-ee5d berlin object "C"  (GeneralStateTests/stCreateTest/CREATE2_CallData, step: GeneralStateTestFiller)
Error: Bytecode is missing! ':yul berlin object "C" {
  code {
    let s := datasize("initcode")
    let o := dataoffset("initcode")
    codecopy(0, o, s)
    let r := create2(0, 0, s, 0)
    sstore(0, r)
    stop()
  }

  object "initcode" {
    code {
      sstore(0, calldataload(0))
      calldatacopy(0, 0, 64)
      return(0, msize())
    }
  }
}
' CREATE2_CallData (GeneralStateTests/stCreateTest/CREATE2_CallData, step: GeneralStateTestFiller)

--------
```

Reply:
aha, you found more json bugs.

Reply:
here https://github.com/chfast/tests/pull/1

Reply:
ok, thanks a lot. I just pulled your changes.